src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseBackupTests.cs

```diff
@@ -80,7 +80,7 @@ public void TestServerBackupLongTermRetentionVault()
                 RunPowerShellTest("Test-ServerBackupLongTermRetentionVault");
             }
         }
-        [Fact]
+        [Fact(Skip = "LTR-V1 restore service is retiring in Prod.")]
```

Comment:
@jaredmoo @akromm should we remove this test if the service is being retired? Also, how does this affect the corresponding cmdlets?